### DISCUSSION TITLE
Ports used for the bitwarden extension

### DISCUSSION THREAD

**Discussion by tweetysat** [dani-garcia/vaultwarden#3123] [Q&A] [answered]
I'm using docker compose for starting my container with something like

    vaultwarden:
        image: vaultwarden/server:latest
        container_name: vaultwarden
        restart: unless-stopped
        volumes:
            - "data:/data/"
        ports:
            - 80:80

I can then access to the ui with http://xxxx.com and create an account.
Than I install the chrome extension and put http://xxxx.com in the servers' url.
I can now login and access to my vault.

But there's something I don't understand : if I remove the ports in my docker-compose.yml and restart the container, I cannot access anymore the ui in my browser.  Ok it's normal.  But I can still access my vault within the extension (but if I disconnect I cannot reconnect again).  How is it possible ?

**Comment by BlackDex** [comment, marked as answer]
If you do not add the port config docker will not create a proxy/forward for it.

The extension is probably only using is cache and it's not able to sync for new data.

**Reply by tweetysat** [reply]
Ok that seems to right explanation.
I can access my secrets, I can add new ones but I cannot synchronize with the server.
So it means the extension is caching all the secrets ! Where is the cache located in, let's say, chrome ?

**Reply by BlackDex** [reply]
I'm not sure where the cache is, but it would be within the browser indeed. In it's local storage i guess

**Reply by tweetysat** [reply]
Ok thanks.
